cli/waiter/token_post.py

```diff
@@ -43,18 +44,21 @@ def post_failed_message(cluster_name, reason):
     return f'Token post {terminal.failed("failed")} on {cluster_name}:\n{terminal.reason(reason)}'
 
 
-def create_or_update(cluster, token_name, token_fields, action):
+def create_or_update(cluster, token_name, token_fields, action, update_mode_admin=False):
```

Comment:
Can we rename the variable to `admin_mode`? 
Also, I'd prefer it to not have a default value and be placed before `action` in the parameter list.